stories/navbar/props-table.tsx

```diff
@@ -17,6 +17,12 @@ const DATA: ApiTableSectionProps[] = [
                     "Navbar resources such as the brand name and logo source",
```

Comment:
I think we've mentioned previously that the first PR should cover the component and a basic story first. Stuff like props table should be a separate PR after we're happy with the props and component